app/index.html

```diff
@@ -43,6 +43,24 @@
 </head>
 
 <body>
+	<noscript>
+		<!--
+			This will be displayed if the user doesn't have JavaScript enabled. 	
+		-->
+		LLDAP requires JavaScript, please switch to a compatible browser or enable 
+		it.
+	</noscript>
+
+	<script>
+		/* Detect if the user has WASM support. */
+		if (typeof WebAssembly === 'undefined') {
+			const pWASMMsg = document.createElement("p")
+			pWASMMsg.innerHTML = `
+			LLDAP requires WASM, please switch to a compatible browser or enable it.
```

Comment:
For the users of brave (or derived browsers), it might require enabling JIT for JavaScript. Could we add a note about that?

Reply:
I've added JIT to the message.